internal/options/options.go

```diff
@@ -67,6 +73,8 @@ func (o *Options) AddFlags() {
 	o.flags.StringVar(&o.ExporterHost, "exporter-host", "0.0.0.0", "Host to expose kube-events-exporter own metrics on.")
 	o.flags.IntVar(&o.ExporterPort, "exporter-port", 8081, "Port to expose kube-events-exporter own metrics on.")
 	o.flags.BoolVar(&o.Version, "version", false, "kube-events-exporter version information")
+
+	o.flags.StringArrayVar(&o.EventTypes, "event-types", []string{EventTypesAll}, "Event types filter. Defaults to all types.")
```

Comment:
We should document later on all the various options, possibly by pointing out to Events docs on this, I would not add it to the flag, just note on this needs to be well documented.